setup.py

```diff
@@ -4,7 +4,7 @@
     long_description = fh.read()
 
 setup(name='jsons',
-      version='0.5.4',
+      version='0.5.5',
```

Comment:
Since you've added a nice feature as well, I think we should increase the version as follows:

```suggestion
      version='0.6.0',
```